ledger/src/ui/test/onboard/test_onboard.c

```diff
@@ -76,8 +76,11 @@ unsigned int bolos_ux_mnemonic_from_data(unsigned char *in,
                                          unsigned int inLength,
                                          unsigned char *out,
                                          unsigned int outLength) {
-    sprintf((char *)out, "mnemonics-generated-from-%s", in);
-    return strlen((const char *)out);
+    int len = strlen("mnemonics-generated-from-");
```

Comment:
We should also bear in mind that string literals in C are automatically appended a null-terminator.